src/site/asn1.rst

```diff
@@ -69,6 +69,36 @@ Part 2 --- Country Catalogue
 ============================
 
 
+#. Create a method called ``setPopulation`` that takes a name of a country and a new population value for that country as parameters
+
+#. Have a method called ``numberCountriesOnContinent`` that takes a continent name as a parameter and returns the number of ``Country`` objects on that continent
+
+#. Write a method called ``filterCountriesByContinent`` that takes a continent name as a parameter and returns a ``Country`` array containing only the countries on the specified continent
+    * You may find ``numberCountriesOnContinent`` useful here
```

Comment:
I was considering this to be returning a new ``CountryCatalogue`` as opposed to an array of ``Country`` objects.

Reply:
An array constructor is simple enough I think to add it. Implementation is just `for each country { add(country) }`. Such a constructor would make the signature a little more sane, and not really be any harder imo.

Reply:
Given the reworking of the constructor in #414, this will be trivial to adjust now.